config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -480,8 +482,105 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
+
+##########################################################
+#  listing-scope: financiering partners
+##########################################################
+ext:partnerListingS a form:Scope;
+  sh:path ext:financingPartner.
+
+##########################################################
+#  Generator: partner
+##########################################################
+ext:partnerGenerator a form:Generator;
+  form:prototype [
+    form:shape [
+      a ext:FinancingPartner ;
+      ext:financingAmount 0;
+    ]
+  ];
+  form:dataGenerator form:addMuUuid.
+
+##########################################################
+#  Listing: Partners
+##########################################################
+ext:partnerListing a form:Listing;
+  a form:ListingTable;
+  form:each ext:partnerFormItem;
+  form:scope ext:partnerListingS;
+  form:createGenerator ext:partnerGenerator;
+  form:canAdd true;
+  form:addLabel "Voeg een nieuwe partner toe";
+  form:canRemove true;
+  form:removeLabel "Verwijder";
+  form:noDataMessage "Er werden nog geen partners toegevoegd";
+  sh:group ext:financingPropertyGroup;
+  sh:order 1 .
+
+##########################################################
+#  Subform: Partner
+##########################################################
+ext:partnerFormItem a form:SubForm;
+   form:includes ext:partnerTitleF;
+   form:includes ext:partnerTypeF;
+   form:includes ext:financingAmountF;
+   form:hasFieldGroup ext:financingPropertyGroup.
+
+##########################################################
+# Field: Titel
+##########################################################
+ext:partnerTitleF a form:Field ;
+    sh:name "Titel" ;
+    sh:order 1;
+    sh:path ext:partnerTitle;
+    form:displayType displayTypes:defaultInput ;
+    form:validations
+      [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:resultMessage "Dit veld is verplicht."@nl;
+        sh:path ext:partnerTitle] ;
+    sh:group ext:financingPropertyGroup .
+
+##########################################################
+# Field: Partnertype
+##########################################################
+ext:partnerTypeF a form:Field ;
+    sh:name "Type partner" ;
+    sh:order 2;
+    sh:path ext:partnerType;
+    form:displayType displayTypes:conceptSchemeSelector ;
+    form:options """{"conceptScheme":"http://lblod.data.gift/vocabularies/subsidie/FinancieringPartnerType","searchEnabled":false}""" ;
+    form:validations
+      [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:resultMessage "Dit veld is verplicht."@nl;
+        sh:path ext:partnerType] ;
+    sh:group ext:financingPropertyGroup .
+
+##########################################################
+# Field: Bedrag
+##########################################################
+ext:financingAmountF a form:Field ;
+    sh:name "Bedrag" ;
+    sh:order 3;
+    sh:path ext:financingAmount;
+    form:displayType displayTypes:numericalInput ;
```

Comment:
This needs to be replaced with something better, but I don't think we have a "currency" field yet?

Reply:
I dont think so, I also just used numericalInput